README.md

```diff
@@ -31,26 +31,35 @@ Require this package with composer using the following command:
 composer require --dev barryvdh/laravel-ide-helper
 ```
 
-If you are using Laravel versions older than 5.5, add the service provider to the `providers` array in `config/app.php`:
```

Comment:
Since we only support 5.5+, it's not necessary to refer to older versions.

As auto-discovery thus is the standard, I rephrased the installation section based on this and added more details how to manually do this in case it's desired